app/transformers/rural-payments-portal/customer.js

```diff
@@ -73,3 +75,24 @@ export function transformNotificationsToMessages (notifications = [], showOnlyDe
     }))
     .filter(({ archivedAt }) => showOnlyDeleted ? archivedAt !== null : archivedAt === null)
 }
+
+export function transformPersonSummaryToCustomerAuthorisedFilteredBusiness (sbi, summary) {
+  const filteredBusinessForCustomer = summary.filter(person => person.sbi === sbi)
+  if (filteredBusinessForCustomer.length === 0) {
+    throw new GraphQLError(
+      'Customer does not have a permission to view business information',
+      {
+        extensions: {
+          code: 'FORBIDDEN'
+        }
+      }
+    )
+  }
+
+  return {
+    id: filteredBusinessForCustomer[0].id,
+    customerId: filteredBusinessForCustomer[0].id,
```

Comment:
```suggestion
    customerId: filteredBusinessForCustomer[0].customerId,
```

`id` and `customerId` aren't the same

Reply:
there is no customer id in summary in original
```
return summary.map(({ id, name, sbi }) => ({
    customerId,
    id,
    name,
    sbi
  }))

```
and in parsed you have this: 

```
async businesses ({ id }, __, { dataSources }) {
    const summary = await dataSources.ruralPaymentsPortalApi.getPersonSummaryByPersonId(id)
    return transformPersonSummaryToCustomerAuthorisedBusinesses(id, summary)
  }
```

Reply:
done in commit: 11bd36b